src/dotnet/commands/dotnet-new/CSharp_xunittest/project.json.template

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24008",
+        "xunit": "2.1.0",
+        "dotnet-test-xunit": "1.0.0-dev-140469-38"
```

Comment:
our tests in the CLI repo are using:

`"dotnet-test-xunit": "1.0.0-rc2-192208-24"`